app/views/menus/_admin_tabs.haml

```diff
@@ -3,8 +3,13 @@
     - if can_edit_users?
       %li.nav-item{role: "presentation", class: ('active' if current_page?(admin_users_path))}
         = link_to 'Users', admin_users_path, class: 'nav-link'
+    - if can_edit_roles?
       %li.nav-item{role: "presentation", class: ('active' if current_page?(admin_roles_path))}
         = link_to 'Roles', admin_roles_path, class: 'nav-link'
+    - if can_edit_access_groups?
```

Comment:
Same here, this permission should be on the ability to create/delete groups, not on the assignment of them to users.

Reply:
Same reply :-)